src/components/streetplan-loader.js

```diff
@@ -78,6 +78,8 @@ AFRAME.registerComponent('streetplan-loader', {
       return;
     }
 
+    STREET.sourceType = 'streetplanURL'; // it also could be jsonFile/streetmixURL
```

Comment:
@Algorush there will be more than 1 street per scene so I don't understand why this needs to be set ?

Reply:
Yes, I thought about more than 1 street per scene and was tested with this case also. `STREET.sourceType` refers to the currently loaded street. 
And yes, maybe its not good idea, but I didn’t find a better option when was thinking about it.